hardhat.config.ts

```diff
@@ -40,11 +40,6 @@ const config: HardhatUserConfig = {
     },
   },
   networks: {
-    sepolia: {
-      url: process.env.SEPOLIA_URL || '',
-      chainId: 11155111,
-      accounts: process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
-    },
```

Comment:
Why did you remove `sepolia`?

Reply:
I think I just moved it down.

Reply:
For some reason, the only change in diff was removing it, but I checked the file, and it is there. We are good to go! 🚀